lib/Service/SettingsService.php

```diff
@@ -112,6 +112,9 @@ public function set(string $uid, array $settings, bool $writeDefaults = false) :
 			if ($value !== null && array_key_exists($name, $this->attrs)) {
 				$settings[$name] = $value = $this->attrs[$name]['validate']($value);
 			}
+			if ($name === 'notesPath' && $value !== null) {
```

Comment:
Do we need to limit this to the notesPath or could we just skip getting the default if the value is not null?